apps/api/src/queries/presentation/formatAggregateType.ts

```diff
@@ -32,7 +31,7 @@ export default (aggregateType: AggregateType): string => {
     const label = resourceTypeAndLabel[aggregateType];
 
     if (!isNonEmptyString(label)) {
-        throw new InternalError(`Failed to find label for resource type: ${aggregateType}`);
+        return aggregateType;
```

Comment:
Add a comment stating that we default to this instead of throwing to support dynamic resource types, which is currently important for loosely coupled tests.